### DISCUSSION TITLE
GitHub Actions stopped telling the reason for job cancellation on timeout

### DISCUSSION THREAD

**Discussion by yeputons** [community/community#40582] [Actions]
### Select Topic Area

Product Feedback

### Body

In the fall of 2021, if you had a job which timed out, you was able to understand that by going on workflow's Summary page, scrolling down to "Annotations" and seeing text like `The job running on runner ..... has exceeded the maximum execution time of 3 minutes.`.

Nowadays the comment lacks any details about the timeout and only says `The operation was canceled.` without specifying the exact reason. The only way to _guess_ (not know for sure) is to check job's running time and compare it with workflow's configuration.

That's very confusing, especially for people who are just getting started with GitHub Actions. You have to both know about that quirk and be able to look up the timeout somewhere in the settings or the docs.

Would be nice if we can have that behavior back.

**Comment by ossanna16** [comment]
Hi there @yeputons and welcome to our community! Thank you for the feedback!

**Comment by yeputons** [comment]
It seems that this behavior was reverted at some point. Thank you!

Here is a screenshot from three months ago, the `MSVC, Debug` job timed out and cancelled the rest:

![image](https://user-images.githubusercontent.com/1297325/224570649-5147466f-651f-4413-994d-6de06d451f66.png)

Here is a screenshot from today, the `GCC, Valgrind, Debug` job timed out and cancelled the rest:

![image](https://user-images.githubusercontent.com/1297325/224570731-2c0a04dc-8839-4412-bf71-a30dfb39c2fd.png)
